src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,24 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
+            logger.info("Attempting to login");
```

Comment:
Let's make our log messages informative. Include parameters in the log message to make it more useful for debugging purposes. For example, you could log the username attempting to login.